runtime/parachains/src/configuration.rs

```diff
@@ -1030,9 +1008,20 @@ pub mod pallet {
 			new: T::BlockNumber,
 		) -> DispatchResult {
 			ensure_root(origin)?;
-			Self::update_config_member(|config| {
+			Self::schedule_config_update(|config| {
 				config.minimum_validation_upgrade_delay = new;
-			});
+			})
+		}
+
+		/// Setting this to true will disable consistency checks for the configuration setters.
```

Comment:
```suggestion
		/// Setting this to true will disable consistency checks for the configuration setters.
		///
```

Reply:
I actually wanted to leave it in the summary.